### PR TITLE
Functional tests - Add movement check after update stocks quantity

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Update test `Update Quantity' : add check of product name and quantity in movement after stock update
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | `TEST_PATH="functional/BO/03_catalog/08_stocks/02_updateQuantity.js" URL_FO=SHOP_URL npm run specific-test`

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17710)

### CODE REVIEW

File: tests/puppeteer/pages/BO/catalog/stocks/index.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ module.exports = class Stocks extends BOBasePage {
     this.successfulUpdateMessage = 'Stock successfully updated';
 
     // Selectors
-    this.MovementNavItemLink = '#head_tabs li:nth-child(2) > a';
+    this.MovementsNavItemLink = '#head_tabs li:nth-child(2) > a';
```
Comment (Progi1984) [inline tests/puppeteer/pages/BO/catalog/stocks/index.js]:
```suggestion
    this.movementsNavItemLink = '#head_tabs li:nth-child(2) > a';
```

**Review by Progi1984 — COMMENTED** [review]
Small feedback

File: tests/puppeteer/pages/BO/catalog/stocks/movements/index.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        throw new Error(`${column} was not find as column in this table`);
+    }
+  }
+
+  /**
+   * Get number of element in movements grid
+   * @return {Promise<integer>}
+   */
+  async getNumberOfElementInGrid() {
+    return (await this.page.$$(this.tableRows)).length;
```
Comment (NeOMakinG) [inline tests/puppeteer/pages/BO/catalog/stocks/movements/index.js:79]: Is the double $$ normal ?

File: tests/puppeteer/pages/BO/catalog/stocks/index.js (hunk 1 above)
Comment (boubkerbribri) [reply tests/puppeteer/pages/BO/catalog/stocks/index.js]: nice catch, done

File: tests/puppeteer/pages/BO/catalog/stocks/movements/index.js (hunk 2 above)
Comment (boubkerbribri) [reply tests/puppeteer/pages/BO/catalog/stocks/movements/index.js:79]: yes, we use $$ to get array of elements and $ to get the first element. you can find the doc [here](https://pptr.dev/#?product=Puppeteer&version=v1.9.0&show=api-pageselector-1)

**Review by NeOMakinG — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Pull request merged**